recipes/python-toolchain/bld.bat

```diff
@@ -0,0 +1,16 @@
+setlocal EnableDelayedExpansion
+
```

Comment:
This afaik only is useful when you use `!VAR!`instead of `%VAR%`. Also as far as I can see, there is no case below which would need delayed variables: in the for loop it's either a loop variable or a variable which was defined outside of the loop.